sleap/gui/app.py

```diff
@@ -1166,6 +1166,10 @@ def _update_gui_state(self):
         self.overlays["track_labels"].visible = (
             control_key_down and has_selected_instance
         )
+        
+        # Update session state
+        if (has_video):
+            self.state["session"] = self.labels.get_session(self.state["video"])
```

Comment:
I was a bit skeptical, but honestly this might just work. I was worried this might cause problems for the future if we decide to add a callback(s) for whenever the `"session"` state is updated, but it looks like `GuiState` has a custom `__setitem__` method that ensures the the new value is actually new:
https://github.com/talmolab/sleap/blob/0134ab443663bf1b1f0287d632dce9fdef80c14b/sleap/gui/state.py#L56-L61

However, doing things this way, although easier, will be more expensive than setting once upon video state change (`_update_gui_state` is run as a background process and called every few ms):
https://github.com/talmolab/sleap/blob/0134ab443663bf1b1f0287d632dce9fdef80c14b/sleap/gui/app.py#L309-L312

So! I would prefer if we [searched the repo for all places we set `state["video"] =`](https://github.com/search?q=repo%3Atalmolab%2Fsleap%20state%5B%22video%22%5D%20%3D&type=code) and ensure to also set the `state["session"]` in those places instead of the current implementation above.